app/views/instrument_schedule_positions/show.html.haml

```diff
@@ -0,0 +1,26 @@
+= content_for :h1 do
+  = current_facility
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: "instrument_schedule_position"
+
+%h2= text("title")
```

Comment:
It might be worth adding a little description here of where this impacts the views.